packit/config/aliases.py

```diff
@@ -268,68 +254,22 @@ def get_all_koji_targets() -> list[str]:
     return run_command(["koji", "list-targets", "--quiet"], output=True).stdout.split()
 
 
-def update_metrics():
-    if hasattr(pushgateway, "aliases_fallback_used"):
-        pushgateway.aliases_fallback_used.inc()
-    pushgateway.push()
-
-
 @ttl_cache(maxsize=1, ttl=timedelta(hours=12).seconds)
```

Comment:
I see we are using the cache of the `fedora-distro-aliases`, is this still needed?

Reply:
Ok, will do.